src/components/Components/LinkedSecretsListView/LinkedSecretsListView.tsx

```diff
@@ -0,0 +1,170 @@
+import PageLayout from '../../PageLayout/PageLayout';
+import getListHeader, { SortableHeaders } from './LinkedSecretsListHeader';
+import { LinkedSecretsListRow } from './LinkedSecretsListRow';
+
+const sortPaths: Record<SortableHeaders, string> = {
+  [SortableHeaders.secretName]: 'metadata.name',
+  [SortableHeaders.type]: 'type',
+};
+
+export const LinkedSecretsListView: React.FC = () => {
+  const { componentName, applicationName } = useParams<RouterParams>();
+  const namespace = useNamespace();
+  const applicationBreadcrumbs = useApplicationBreadcrumbs();
+  const [component, componentLoaded, componentError] = useComponent(namespace, componentName);
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number>(SortableHeaders.secretName);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<SortByDirection>(
+    SortByDirection.asc,
+  );
+  const [linkedSecrets, linkedSecretsLoaded, linkedSecretsError] = useLinkedSecrets(
+    namespace,
+    componentName,
+  );
+
+  const LinkedSecretsListHeader = React.useMemo(
+    () =>
+      getListHeader(activeSortIndex, activeSortDirection, (_, index, direction) => {
+        setActiveSortIndex(index);
+        setActiveSortDirection(direction);
+      }),
+    [activeSortDirection, activeSortIndex],
+  );
+
+  const sortedLinkedSecrets = useSortedResources(
+    linkedSecrets,
+    activeSortIndex,
+    activeSortDirection,
+    sortPaths,
+  );
+
+  if (!componentLoaded || !linkedSecretsLoaded) {
+    return (
+      <Bullseye>
+        <Spinner data-test="spinner" />
+      </Bullseye>
+    );
+  }
+
+  if (componentError || linkedSecretsError) {
+    const error = componentError ?? linkedSecretsError;
+    const httpError = HttpError.fromCode(error ? (error as { code: number }).code : 404);
+    return (
+      <ErrorEmptyState
+        data-test="linked-secrets-list-view_error-empty-state"
+        httpError={httpError}
+        title="Unable to load linked secrets"
+        body={httpError?.message.length ? httpError?.message : 'Something went wrong'}
+      />
+    );
+  }
+
+  const EmptyMessage = () => (
+    <FilteredEmptyState
+      onClearFilters={() => {}} // TODO: clear name filter - will be implemented in another ticket
+      data-test="linked-secrets-list-view_filtered-empty-state"
+    />
+  );
+
+  const NoDataEmptyMessage = () => (
+    <AppEmptyState
+      emptyStateImg={emptyStateImgUrl}
+      title="No linked secrets found for this component"
+      data-test="linked-secrets-list-no-data-empty-message"
+    >
+      <EmptyStateBody>To get started, link a new secret.</EmptyStateBody>
+    </AppEmptyState>
+  );
+
+  return (
+    <>
+      <PageLayout
+        title="Manage linked secrets"
+        description={
+          <>
+            You can add new secrets to this component or unlink existing secrets.{' '}
+            <ExternalLink href="">Learn more about linked secrets</ExternalLink>
```

Comment:
we still don't have the link to the docs, we might probably add it later on (maybe in a new ticket??)

suggestion: we could potentially comment out this code, so the user wouldn't be presented with a "link that doesn't work" - let me know what you guys think :)

Reply:
I vote it for 'comment out'. For help info and docs links for the whole feature, i am considering to create one issue to track those all.
We have no ideas when it's ready. To avoid users creating bugs for it, i guess the known issue could be used to address all this kind of problems.

Reply:
agreed! I'm going to comment it out and we can update the code once we know the correct link